ui/StatusQ/src/StatusQ/Popups/StatusMenu.qml

```diff
@@ -57,6 +57,16 @@ Menu {
 
     signal menuItemClicked(int menuIndex)
 
+    function checkNotEmpty() {
```

Comment:
Nit: double negations make my head hurt lol.
Maybe `checkIfEmpty` would be simpler to understand? (don't forget to flip the returns then)

Reply:
fixed